pkg/ipamd/ipamd.go

```diff
@@ -1075,6 +1190,53 @@ func (c *IPAMContext) eniIPPoolReconcile(ipPool []string, attachedENI awsutils.E
 	}
 }
 
+func (c *IPAMContext) eniPrefixPoolReconcile(ipPool []string, attachedENI awsutils.ENIMetadata, eni string) {
+	attachedENIIPs := attachedENI.IPv4Prefixes
+	needEC2Reconcile := true
+	// Here we can't trust attachedENI since the IMDS metadata can be stale. We need to check with EC2 API.
+	log.Infof("Found prefix pool count %d for eni %s\n", len(ipPool), eni)
+	for _, dumpIP := range ipPool {
+		log.Infof("DUMP from eni pool %s", dumpIP)
```

Comment:
`Debugf`  (and elsewhere in this function)

Reply:
I had removed it, merge from internal repo to GitHub messed this up :(